refinery/tool_manager/tests.py

```diff
@@ -1192,12 +1192,10 @@ def test_tool_container_removed_on_deletion(self):
         self.view(post_request)
 
         tool = Tool.objects.get(tool_definition__uuid=td.uuid)
+        tool_uuid = tool.uuid
```

Comment:
I see. Actually, there is no need to test external libraries: `DockerClientWrapper().purge_by_label()` in `remove_tool_container()` should be mocked and then tested if it was called then there is no need to save the tool UUID.